athlinks-schema.ttl

```diff
@@ -158,7 +195,7 @@ athlinks:pace
 athlinks:seriesType
   a owl:DataTypeProperty;
   rdfs:range [ a xsd:string ;
-                owl:oneOf("Master" "Social" "Points" "Series")
+                owl:oneOf("Master" "Social" "Points" "Series" "MasterSeries")
```

Comment:
They want to be able to define that a series occurs every year in the same way that a Master Event defines that an event occurs every year.